### ISSUE TITLE
 Standard names: *Friction Velocity in Sea Water*

### ISSUE THREAD

**Issue by abiardeau** [cf-convention/discuss#316]
Hello,
I am Aurore BIARDEAU from Mercator Ocean international, working for Copernicus Marine service.
Date : 13-05-2024

We are in the process of implementing a new variable in the Copernicus Marine service and we would be happy to have your feedback on our proposal :

**standard_name** : `eastward_friction_velocity_in_sea_water` (same for northward) 
(Inspired by existing standard_name `eastward_friction_velocity_in_air`)

**unit** : m s-1

I can provide more information from my expert colleagues if needed,
Kind regards,
Aurore

**Comment by JonathanGregory** [comment]
Dear Aurore @abiardeau

For friction velocity in sea water, I suppose we ought to specify that it's a _surface_ friction velocity, because the concept applies to the sea floor as well.

Best wishes

Jonathan

**Comment by JonathanGregory** [comment]
Yes, `at_sea_water_surface` is a phrase we've used before, so that would work. Another possibility is `surface_eastward_friction_velocity_in_sea_water`.

**Comment by abiardeau** [comment]
Thanks, both would work for us. If there is any preference from your side we we'll go with it

**Comment by efisher008** [comment]
Dear Aurore,

I have added your proposed name `eastward_friction_velocity_in_sea_water` to the editor, and will wait for more discussion on the format before proceeding (including the change to one of the above suggestions). It would be useful to have a description for this name to help clarify its use. 

Best regards,
Ellie

**Comment by efisher008** [comment]
Hello Aurore @abiardeau,

Looking back at this issue, I am wondering if it would be more descriptive to use the form `eastward_friction_velocity_at_sea_water_surface`/`northward_friction_velocity_at_sea_water_surface` for the names. This would specify that it is a **surface** quantity as @JonathanGregory had mentioned, and there is some precedent in the standard names table for using `at_sea_water_surface`.

If you could provide a description for the names, that would be very useful. I have added the description for eastward (_"Eastward" indicates a vector component which is positive when directed eastward (negative westward)."_) and northward (_""Northward" indicates a vector component which is positive when directed northward (negative southward)."_) to them as a starting point. 

Best wishes,
Ellie

**Comment by efisher008** [comment]
Hi Aurore @abiardeau,

I hope your summer is going well. 
Now that your other batch of proposed standard names have been accepted, would you have any time to dedicate to progressing this proposal? It would be great to see it moving towards acceptance. 

Best,
Ellie
